docs/rules/eqeqeq.md

```diff
@@ -1,4 +1,6 @@
-# Require === and !== (eqeqeq)
+# eqeqeq
+
+Requires the use of === and !==.
```

Comment:
This one caught my eye. 

```suggestion
Requires the use of `===` and `!==`.
```

Reply:
This seems inconsistent throughout the docs, sometimes technical keywords are highlighted with code sometimes not.

Reply:
Heh, yeah. I'll take a pass through the rules and fix up what I can find.